plugins/rollbar-backend/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@backstage/plugin-rollbar-backend",
+  "version": "0.1.1-alpha.12",
```

Comment:
Doh, looks like I can't solve these conflicts for you. The version here + deps needs to be bumped to alpha.13, and same with the frontend plugin. Search/replace alpha.12 -> alpha.13 should do the trick